src/main/kotlin/lotto/ui/ManualNumberInputView.kt

```diff
@@ -0,0 +1,29 @@
+package lotto.ui
+
+import lotto.data.LottoNumbers
+
+object ManualNumberInputView {
+
+    fun askManualInput(): List<LottoNumbers> {
+        println("수동으로 구매할 로또 수를 입력해 주세요.")
+        val manualTicketCount = askManualTicketCount()
+
+        println("수동으로 구매할 번호를 입력해 주세요.")
+        val manualNumbersList = mutableListOf<LottoNumbers>()
+        repeat(manualTicketCount) {
+            val lottoNumbers = readSingleLottoNumbers()
+            manualNumbersList.add(lottoNumbers)
+        }
+        return manualNumbersList
+    }
+
+    private fun askManualTicketCount(): Int {
+        val manualTicketNumInput = readLine()
+        return manualTicketNumInput?.toIntOrNull() ?: throw IllegalArgumentException("잘못된 입력값이 들어왔습니다.($manualTicketNumInput)")
+    }
```

Comment:
잘못된 입력값이 들어오면 다시 입력을 받아보는 것은 어떨까요?

```suggestion
    private tailrec fun askManualTicketCount(): Int {
        println("수동으로 구매할 로또 수를 입력해 주세요.")
        return readLine()?.toIntOrNull() ?: askManualTicketCount()
    }
```

`tailrec`은 꼬리 재귀 최적화를 위한 키워드입니다.